tests/Support/Constraint/CssConstraint.php

```diff
@@ -167,8 +167,17 @@ abstract class CssConstraint extends Constraint
      */
     protected static function getCssRegularExpressionMatcher(string $css): string
     {
-        $callback = \Closure::fromCallable([self::class, 'getCssRegularExpressionReplacement']);
-        $matcher = (new Preg())->replaceCallback(self::CSS_REGULAR_EXPRESSION_PATTERN, $callback, $css);
+        $callback = static function (array $matches): string {
+            /** @var array<int<0, max>, string> $matches $matches */
```

Comment:
Duplicate `$matches`.

```suggestion
            /** @var array<int<0, max>, string> $matches */
```